server/odc-service/src/main/java/com/oceanbase/odc/service/task/service/StdTaskFrameworkService.java

```diff
@@ -154,6 +154,14 @@ public Page<JobEntity> findHeartTimeTimeoutJobs(int timeoutSeconds, int page, in
         return page(condition, page, size);
     }
 
+    @Override
+    public Page<JobEntity> findNotTerminalJobs(int page, int size) {
+        Specification<JobEntity> condition = Specification.where(getRecentDaySpec(RECENT_DAY))
```

Comment:
Only querying the data of the past 30 days may miss published tasks, which may be optimized in the future.

Reply:
ok, this wil be optimized by archive history data